Chapter-1-initial-architecture/Src/Fitnet.UnitTests/Contracts/SignContract/SignContractTestData.cs

```diff
@@ -2,30 +2,26 @@ namespace EvolutionaryArchitecture.Fitnet.UnitTests.Contracts.SignContract;
 
 internal sealed class SignContractTestData : IEnumerable<object[]>
```

Comment:
I adjusted it in chapter 2 to a bit different form - can you please adjust it here as well?

Reply:
I just recall that for such case is cleaner why that we have currently implemented in chapter 2.
`internal sealed class SignContractTestData : TheoryData<DateTimeOffset, DateTimeOffset, DateTimeOffset, DateTimeOffset>
{
    public SignContractTestData()
    {
        AddRow(new DateTimeOffset(2021, 1, 1, 1, 1, 1, TimeSpan.Zero),
            new DateTimeOffset(2021, 1, 1, 1, 1, 1, TimeSpan.Zero),
            new DateTimeOffset(2021, 1, 1, 1, 1, 1, TimeSpan.Zero),
            new DateTimeOffset(2022, 1, 1, 1, 1, 1, TimeSpan.Zero));

        AddRow(new DateTimeOffset(2022, 6, 1, 12, 0, 0, TimeSpan.Zero),
            new DateTimeOffset(2022, 6, 1, 12, 0, 0, TimeSpan.Zero),
            new DateTimeOffset(2022, 6, 1, 12, 0, 0, TimeSpan.Zero),
            new DateTimeOffset(2023, 6, 1, 12, 0, 0, TimeSpan.Zero));
        
        AddRow(new DateTimeOffset(2023, 2, 15, 8, 30, 0, TimeSpan.Zero),
            new DateTimeOffset(2023, 2, 15, 8, 30, 0, TimeSpan.Zero),
            new DateTimeOffset(2023, 2, 15, 8, 30, 0, TimeSpan.Zero),
            new DateTimeOffset(2024, 2, 15, 8, 30, 0, TimeSpan.Zero));
    }
}`

Reply:
@meaboutsoftware what you think. I could adjust in all chapter to this approach?